packages/common/src/getByteLength.ts

```diff
@@ -0,0 +1,6 @@
+import { Hex } from "viem";
+
+export function getByteLength(data: Hex): number {
```

Comment:
viem has `size` for this purpose: https://github.com/wagmi-dev/viem/blob/main/src/utils/data/size.ts#L12

Reply:
oh nice, i figured there must exist something for a super general util like this one